src/features/dashboard/DashboardPage.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import Dots from '../../common/components/Icons/Dots'
+import TransactionItem from './TransactionItem'
+import PortfolioColumn from './PortfolioColumn'
+import PortfolioItem from './PortfolioItem'
+import NFTCard from './NFTCard'
+import Notification from './Notification'
+
+const transactions: { type: 'in' | 'out'; amount: string }[] = [
+  { type: 'in', amount: '320,000 PSL' },
+  { type: 'out', amount: '123,000 PSL' },
+  { type: 'in', amount: '320,000 PSL' },
+  { type: 'out', amount: '123,000 PSL' },
+  { type: 'in', amount: '320,000 PSL' },
+  { type: 'out', amount: '123,000 PSL' },
+  { type: 'in', amount: '320,000 PSL' },
+]
+
+const notifications = [
+  '1 new listing',
+  '1 new sale',
+  '1 new transfer',
+  '1 new sale',
+  '1 new listing',
+  '1 new sale',
+  '1 new transfer',
+  '1 new sale',
+]
```

Comment:
@CrazyOptimist I think we can refactor dashboard.tsx file, now it's a bit tricky to understand. I am not 100%, but it's good to refactor more in-depth, i.e this can move to notification.tsx etc, if so, it will be simple in dashboard.
```
<dashboard>
...
<NFTCard/>
<Notification/>
...
</dashboard>

Reply:
got it